examples/todoapp/app.py

```diff
@@ -49,12 +51,12 @@ def post(request: Request) -> t.List[t.Dict["str", "str"]]:
     return "Item added."
 
 
-@view_config(name="openapi_validation_error")
+@exception_view_config(RequestValidationError, renderer="json")
 def openapi_validation_error(
-    context: DefaultRootFactory, request: Request
+    context: HTTPException, request: Request
```

Comment:
context: https://docs.pylonsproject.org/projects/pyramid/en/latest/glossary.html#term-context is a bit special word used in Pyramid, **exception**  is totally fine

Reply:
Didn't want to change original code here.

Reply:
I, see it's event implied in https://github.com/Pylons/pyramid/blob/master/src/pyramid/httpexceptions.py#L1326 that it can be both.

As it's not consistent anyway we can ignore it.